templates/single.html

```diff
@@ -13,16 +13,10 @@
 <!-- /wp:group --></div>
 <!-- /wp:group -->
 
-<!-- wp:group {"align":"full","style":{"spacing":{"margin":{"bottom":"var:preset|spacing|40"}}},"layout":{"type":"default"}} -->
-<div class="wp-block-group alignfull" style="margin-bottom:var(--wp--preset--spacing--40)">
+<!-- wp:post-content {"lock":{"move":false,"remove":true},"align":"full","layout":{"type":"constrained"}} /-->
```

Comment:
```html
<!-- wp:post-content {"lock":{"move":false,"remove":true},"align":"full","layout":{"type":"constrained"}} /-->
```

can be further improved to

```html
<!-- wp:post-content {"lock":{"move":false,"remove":true},"layout":{"type":"constrained"}} /-->
```

Reply:
This works on WP 6.4-beta2 with Gutenberg 16.7.1.

The parent here is `main.wp-block-group` with default (not constrained) layout.

Full/wide alignment only works within a constrained layout container, so having it here is redundant and editor does not even display controls to set/change the alignment.